test/token/ERC721/extensions/ERC721Votes.test.js

```diff
@@ -1,181 +1,197 @@
-/* eslint-disable */
-
-const { expectEvent, time } = require('@openzeppelin/test-helpers');
+const { ethers } = require('hardhat');
 const { expect } = require('chai');
+const { loadFixture, mine } = require('@nomicfoundation/hardhat-network-helpers');
 
-const { clock, clockFromReceipt } = require('../../../helpers/time');
+const { bigint: time } = require('../../../helpers/time');
 
 const { shouldBehaveLikeVotes } = require('../../../governance/utils/Votes.behavior');
 
-const MODES = {
-  blocknumber: artifacts.require('$ERC721Votes'),
+const TOKENS = [
+  { Token: '$ERC721Votes', mode: 'blocknumber' },
   // no timestamp mode for ERC721Votes yet
-};
+];
+
+const name = 'My Vote';
+const symbol = 'MTKN';
+const version = '1';
+const tokens = [10000000000000000000000000n, 10n, 20n, 30n];
```

Comment:
This number is just too long

```suggestion
const tokens = [ethers.parseEther('10000000'), 10n, 20n, 30n];
```